var/spack/repos/builtin/packages/hdf-eos5/package.py

```diff
@@ -46,6 +46,11 @@ class HdfEos5(AutotoolsPackage):
     conflicts('~static', when='~shared',
               msg='At least one of +static or +shared must be set')
 
+    # payerle: I am adding myself as a maintainer because I initially wrote
+    # the spack recipe, but I am not really an user of the code, so please
+    # consider becoming a maintainer if you use this code.
```

Comment:
I think this is unnecessary as a comment in the package. Maybe when somebody modifies the package you can comment in this direction in the PR?